tests/lib/rules/defaults-on-top.js

```diff
@@ -44,3 +45,16 @@ eslintTester.run("defaults-on-top", rule, {
         }
     ]
 });
+
+eslintTester = new RuleTester({
```

Comment:
I think it's easier to just have one RuleTester with below `parserOptions` they shouldn't conflict with existing tests. And creating multiple instance of RuleTester is a bit confusing.